### ISSUE TITLE
l4d2_drop_secondary.smx: Native "SDKHooks_DropWeapon" reported: Weapon 112 is not owned by client 6

### ISSUE THREAD

**Issue by rpdelaney** [Attano/Equilibrium#9]
```
L 02/26/2014 - 13:53:40: SourceMod error session started
L 02/26/2014 - 13:53:40: Info (map "c3m3_shantytown") (file "errors_20140226.log")
L 02/26/2014 - 13:53:40: [SM] Native "SDKHooks_DropWeapon" reported: Weapon 112 is not owned by client 6
L 02/26/2014 - 13:53:40: [SM] Displaying call stack trace for plugin "optional/l4d2_drop_secondary.smx":
L 02/26/2014 - 13:53:40: [SM]   [0]  Line 46, C:\Users\Mr. DeWitt\Documents\GitHub\Equilibrium\source\l4d2_drop_secondary.sp::PlayerDeath()
L 02/26/2014 - 13:53:42: [SM] Native "SDKHooks_DropWeapon" reported: Invalid entity index -1 for weapon
L 02/26/2014 - 13:53:42: [SM] Displaying call stack trace for plugin "optional/l4d2_drop_secondary.smx":
L 02/26/2014 - 13:53:42: [SM]   [0]  Line 46, C:\Users\Mr. DeWitt\Documents\GitHub\Equilibrium\source\l4d2_drop_secondary.sp::PlayerDeath()
L 02/26/2014 - 14:23:19: Error log file session closed.
```

**Comment by Attano** [comment]
Fixed with 2.1

**Comment by Owned67** [comment]
Hello, sorry for fump but can you upload 2.1 version?

error:
L 06/11/2015 - 10:20:40: SourceMod error session started
L 06/11/2015 - 10:20:40: Info (map "c1m1_hotel") (file "errors_20150611.log")
L 06/11/2015 - 10:20:40: [SM] Native "SDKHooks_DropWeapon" reported: Weapon 447 is not owned by client 3
L 06/11/2015 - 10:20:40: [SM] Displaying call stack trace for plugin "optional/l4d2_drop_secondary.smx":
L 06/11/2015 - 10:20:40: [SM]   [0]  Line 46, C:\Users\Mr. DeWitt\Documents\GitHub\Equilibrium\source\l4d2_drop_secondary.sp::PlayerDeath()
L 06/11/2015 - 10:20:57: [SM] Native "SDKHooks_DropWeapon" reported: Weapon 58 is not owned by client 1
L 06/11/2015 - 10:20:57: [SM] Displaying call stack trace for plugin "optional/l4d2_drop_secondary.smx":
L 06/11/2015 - 10:20:57: [SM]   [0]  Line 46, C:\Users\Mr. DeWitt\Documents\GitHub\Equilibrium\source\l4d2_drop_secondary.sp::PlayerDeath()
L 06/11/2015 - 10:23:29: Error log file session closed.
